### PR TITLE
If being used for Groq, skip OpenAI-Organization header

### PR DESCRIPTION
**Pull request by drnic**
I've looked at using groq via langchain -> ruby-openai and was seeing a warning about me not setting `OpenAI-Organization`. If I run `.compact` on the headers, the warning seems to go away.

### CODE REVIEW

**Comment by drnic** [comment]
^ is this CI failure something I can fix?

**Comment by alexrudall** [comment]
@drnic no, I think I broke something. Will try and fix. Thanks for this PR!

File: .gitignore [hunk 1]
```diff
@@ -14,7 +14,7 @@
 /.yardoc
 /_yardoc/
 /doc/
-
+/TAGS
```
Comment (alexrudall) [inline .gitignore:17]: What's TAGS?

**Comment by alexrudall** [comment]
> ^ is this CI failure something I can fix?

I may have fixed this, if you push another change let's see

File: .gitignore (hunk 1 above)
Comment (drnic) [reply .gitignore:17]: Ugh, it’s one of those metadata files that are spat out by a vscode plugin I have. 

Perhaps there’s a way I can have it in a global ignore on my machine?

Comment (alexrudall) [reply .gitignore:17]: Yeah you can https://stackoverflow.com/questions/45333484/how-to-use-global-gitignore-with-visual-studio-code

Comment (drnic) [reply .gitignore:17]: Cool. You ok to yank this change from the PR? Or I can go find my laptop and do it

**Comment by alexrudall** [comment]
Thanks again @drnic - released in [v7.1](https://github.com/alexrudall/ruby-openai/releases/tag/v7.1.0)

**Pull request merged**